datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxEndpoint.java

```diff
@@ -43,12 +43,24 @@
     @ApiOperation(value = "delete SDX cluster", produces = "application/json", nickname = "deleteSdx")
     void delete(@PathParam("name") String name);
 
+    @DELETE
+    @Path("/crn/{clusterCrn}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "delete SDX cluster by crn", produces = "application/json", nickname = "deleteSdxByCrn")
+    void deleteByCrn(@PathParam("clusterCrn") String clusterCrn);
+
     @POST
     @Path("{name}/redeploy")
```

Comment:
can we use here /name/{name} ? At least this is what we are using in distrox and environment

Reply:
I created a separate JIRA for this because we need to collaborate with qaas guys (@Salaander ) https://jira.cloudera.com/browse/CB-2731